06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,102 @@
+
+/**
+ * InvertedIndex class with constructor
+ * @class
+ */
+class InvertedIndex {
+
+  constructor() {
+    this.indexedFiles = {};
+    this.searchResults = {};
+  }
+
+  /** creates index and update the indexed files
+   * @param {Array} word - object to br returned
+   * @param {Array} filteredContents - an array of all contents in file
+   * @param {object} wordMap - an array of all contents in file
+   * @return  {object}  wordMap - a map of each token to
+   *  there respective indexes
+   */
+  static checkForIndex(word, filteredContents, wordMap) {
```

Comment:
You have 3 functions with JSDoc descriptions saying "creates index and update the indexed files". What's going on? are they all doing the same thing?

Please ensure intentions are clear so I can read through.